tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<p>This may not work well for Colab users. I tried using the below code, which did not work, maybe there is something different to try. If you leave this in, you should make a note that running it locally will work, but maybe not in Colab (or notebooks in general?). I would also mention that <code>save</code> is a method of the folium map so there is no confusion about whether it is associated with GEE.</p><p> </p><pre class="ql-syntax" spellcheck="false">from&nbsp;IPython.display&nbsp;import&nbsp;IFrame
IFrame(src='/content/my_LC_interactive_map.html',&nbsp;width=900,&nbsp;height=600)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>For me, the folium method <em>save </em>works very well and the <em>html</em> file is saved in the content folder of my colab environment. I add this information in the text.</p>